tool/planet/cfg.go

```diff
@@ -79,6 +79,8 @@ type Config struct {
 	ServiceCIDR kv.CIDR
 	// PodCIDR defines the kubernetes Pod subnet CIDR
 	PodCIDR kv.CIDR
```

Comment:
nit: Somewhat unrelated but I faced the same issue with `kv.CIDR` and introduced a local flag that would expose the `net.IPNet` so you don't need to reparse it: https://github.com/gravitational/planet/blob/bernard/6.1.x/subnet-size/tool/planet/args.go#L28. This could save a couple of lines in `verifyPodSubnetSize`.